index.css

```diff
@@ -214,6 +221,58 @@ footer .license {
 	float: right;
 }
 
+footer br{
+	display: none;
+}
+
+h2{
+	text-align: center;
+}
+
+@media only screen and (max-width: 84em){
+	main{	
+		width: 100%;
+		max-width: 57em;
+	}
+
+	footer br{
+		display: visible;
+	}
+
+	footer .license{
+		display: block;
+		float: none;
+		margin-top: .5rem;
+		margin-left: 40%;
+	}
+
+}
```

Comment:
This switches to three columns before three columns can fit into the window. Notice the slight width difference in the following screenshots:
![Screenshot from 2019-10-10 19-10-26](https://user-images.githubusercontent.com/12966757/66590736-b9b5de80-eb91-11e9-874e-8a1fde4018c0.png)
![Screenshot from 2019-10-10 19-10-32](https://user-images.githubusercontent.com/12966757/66590737-b9b5de80-eb91-11e9-9a24-5fe0ed6dd040.png)
![Screenshot from 2019-10-10 19-10-39](https://user-images.githubusercontent.com/12966757/66590738-b9b5de80-eb91-11e9-9f65-b6a55986b30b.png)

Reply:
Are you observing any breakpoints?

Reply:
What do you mean by breakpoints? The screenshots show exactly what I observe.